.eslintrc.yaml

```diff
@@ -235,8 +235,8 @@ rules:
                                       assignment: parens-new-line,
                                       return: parens-new-line,
                                       arrow: parens-new-line,
-                                      condition: parens-new-line,
```

Comment:
Changed these because it would only allow this:
```
{foo
  ? (
      <div>
        <p></p>
      </div>
    )
  : (
      <div>
        <p></p>
      </div>
    )}
```

Instead of this:

```
{foo
  ? <div>
      <p></p>
    </div>

  : <div>
      <p></p>
    </div>}
```

I'm used the the latter style but can revert this if anyone strongly prefers the former.